### PR TITLE
Added test case for series expansion of polynomial/rational expression

### PR DESCRIPTION
**Pull request by abhaydhiman**
#### References to other Issues or PRs

#12791 

#### Brief description of what is fixed or changed
Added a test case for the series expansion of polynomial/rational expression in test_series.py

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/series/tests/test_series.py [hunk 1]
```diff
@@ -294,3 +294,20 @@ def test_issue_11407():
 
 def test_issue_14037():
     assert (sin(x**50)/x**51).series(x, n=0) == 1/x + O(1, x)
+
+
+def test_issue_12791():
+    beta = symbols('beta', reals=True, positive=True)
+    theta, varphi = symbols('theta varphi', reals=True)
```
Comment (sachin-4099) [inline sympy/series/tests/test_series.py:301]: I think this can be `real=True`.

File: sympy/series/tests/test_series.py [hunk 2]
```diff
@@ -294,3 +294,20 @@ def test_issue_11407():
...
+        beta*varphi*sin(theta) - beta*cos(theta) - beta + 1)/(beta*cos(theta) - 1)**2
+
+    sol = 0.5/(0.5*cos(theta) - 1)**2 - 0.25*cos(theta)/(0.5*cos(theta) - 1)**2\
+        + (beta - 0.5)*(-0.5*varphi*sin(theta)*cos(theta)/((0.5*cos(theta) - 1)\
+        **2*(0.5*cos(theta) - 1.0)) - 1/(0.5*cos(theta) - 1)**2 + 0.5*cos(theta)\
+        **2/((0.5*cos(theta) - 1)**2*(0.5*cos(theta) - 1.0)) - 1.0*cos(theta)\
+        /((0.5*cos(theta) - 1)**2*(0.5*cos(theta) - 1.0))) + 0.25*varphi*\
+        sin(theta)/(0.5*cos(theta) - 1)**2 + O((beta - 0.5)**2, (beta, 0.5))
+
+    assert(expr.series(beta, 0.5, 2) == sol)
```
Comment (sachin-4099) [inline sympy/series/tests/test_series.py:313]: I think this can be simply:
`assert expr.series(beta, 0.5, 2) == sol` (without round brackets).

**Comment by sachin-4099** [comment]
Moreover, don't simply add the testcase to the end of the file. Please find issue numbers closer to the current issue number and add the testcase to that place.

**Comment by abhaydhiman** [comment]
So, can I add that in `test_issue_12578` this function?

File: sympy/series/tests/test_series.py (hunk 2 above)
Comment (abhaydhiman) [reply sympy/series/tests/test_series.py:313]: Ok, I'll make the changes suggested by you.

**Comment by sachin-4099** [comment]
> So, can I add that in `test_issue_12578` this function?

No, not inside that. But closer to that.

**Comment by abhaydhiman** [comment]
Just below that function. If I am not wrong.

**Comment by sachin-4099** [comment]
> Just below that function. If I am not wrong.

Yeah should be fine.

**Comment by sachin-4099** [comment]
Looks good. This can be merged @jksuom.

**Comment by abhaydhiman** [comment]
Thanks! for the help @sachin-4099

**Comment by jksuom** [comment]
Thank you, this looks good.

**Pull request merged**